test/e2e/specs/onboarding/vertical__none.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @group calyepso-quarantined
+ */
+
+import {
+	DataHelper,
+	DomainSearchComponent,
+	LoginPage,
+	MyHomePage,
+	NewUserResponse,
+	RestAPIClient,
+	SignupPickPlanPage,
+	StartSiteFlow,
+	UserSignupPage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiCloseAccount } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Vertical: None, Something Else, or Skip' ), function () {
+	const testUser = DataHelper.getNewTestUser( {
+		usernamePrefix: 'free',
```

Comment:
**nit**: Technically, this user would be better prefixed as 'vertical', but this is just nitpick and feel free to ignore what I have said.

Reply:
Good point! I made the change, but I actually didn't end up needing it now that I'm using an existing user instead :)